virttest/guest_agent.py

```diff
@@ -552,6 +552,50 @@ def set_vcpus(self, action):
         return self.cmd(cmd=cmd, args=action)
 
     @error_context.context_aware
+    def get_time(self):
```

Comment:
If command not support, how tester know it, because your always return False when command not exists.
Why not raise exception to told tester command not really support by current guest-agent package.
And better to raise exception when exception occur, don't  delay to resolve  or hidden in test. if did not it, will wast tester's time in analysis.